src/status/installedversion.ts

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
Nitpick - shouldn't this file be called `installedVersion.ts` (with capital `V`)?